_posts/2021-01-15-galaxy-wg.md

```diff
@@ -0,0 +1,24 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+title: 'New Galaxy Community Organization: Working Groups'
+tags: [galaxy]
+---
+
+<br>
+The Galaxy community is undergoing an __organizational transformation__, with the main focus on improving the global coordination in different areas by defining [__Working Groups__](https://galaxyproject.org/community/wg/). The motivations behind this shift are the rapid growth of the community and its diversity.
+
+The Working Groups are open to everyone, with members __distributed across the globe__, and assorted in the topics covered  (admin, backend, UI/UX...). Although their initial definition is set, they might be refined over time.
```

Comment:
Although their initial definition is set, they might be refined over time.

&rarr;

The set of working groups and their missions will evolve over time.